### PR TITLE
NCL-1769 product import start and finish issues

### PR DESCRIPTION
**Pull request by matedo1**

### CODE REVIEW

File: ui/bower.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "font-awesome": "4.2.0",
     "jquery": "2.1.3",
     "angular-resource": "1.3.15",
-    "angular-bootstrap": "0.13.0",
```
Comment (alexcreasy) [inline ui/bower.json]: Out of interest, what does upgrading the angular-bootstrap version fix?

Comment (matedo1) [inline ui/bower.json]: There is new isOpen property for tooltips, you can check it here https://github.com/angular-ui/bootstrap/blob/master/CHANGELOG.md#features-11 , I used this new property on several places like product-import-tree.html:53

Comment (alexcreasy) [inline ui/bower.json]: ahh, yes that is useful!

**Comment by alexcreasy** [comment]
+1 merging!

**Pull request merged**